padevents/padevents.py

```diff
@@ -582,7 +593,9 @@ async def aed_add(self, ctx, server, searchstr, group, time_offset: int = 0):
 
         async with self.config.user(ctx.author).dmevents() as dmevents:
             dmevents.append(default)
-        await ctx.tick()
+
+        await ctx.send("New AED created:")
+        await self.aed_show(ctx, len(dmevents))
```

Comment:
FYI, this isn't *really* a safe thing to do, even if they currently are stored chronologically, they might not be later (e.g. maybe we alphabetize after appending)

I think what I'd do is make a delegated function called _do_add(ctx, whatever, else, you, need) that adds the result and returns the index of the newest one*

then index = _do_add(ctx, ...)

then you can do aed_show(ctx, index) here safely

*yes it might be semi-difficult to actually get that index in a future way of storing them. dw about that now. write safe code now, worry about complicated implementation details of complicated implementations in the future yay